src/dsp/pdr.c

```diff
@@ -523,6 +725,44 @@ pldm_entity_node *pldm_entity_association_tree_add(
 						       false, true, 0xffff);
 }
 
+LIBPLDM_ABI_STABLE
```

Comment:
testing api

Reply:
done

Reply:
i wil change this

Reply:
@vkaverap As discussed in the team channel, lets make it a stable ABI and update dumps as well as part of the PR.